### PR TITLE
enh(Resource-status/Details): Add Graph tab for hosts

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This adds the Graph tab to host details and removes the graph mode from the Services one:

Aug-24-2021 19-15-52

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.10.x (master)

### CODE REVIEW

File: www/front_src/src/Resources/Details/index.test.tsx [hunk 1]
```diff
@@ -1131,13 +1138,12 @@ describe(Details, () => {
...
     act(() => {
       context.setServicesTabParameters({
-        graphMode: false,
         options: defaultGraphOptions,
       });
     });
   });
 
-  it('displays the linked service graphs when the service tab of a host is clicked and the graph mode is activated', async () => {
+  it('displays the linked service graphs when the graph tab of a host is clicked', async () => {
```
Comment (bdauria) [inline www/front_src/src/Resources/Details/index.test.tsx:1146]:
```suggestion
  it('displays the linked service graphs when the Graph tab of a host is clicked', async () => {
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -16222,3 +16222,6 @@ msgstr "Options de recherche"
 
 msgid "The parameter \"%s\" must be an integer"
 msgstr "Le paramètre \"%s\" doit être un entier"
+
+msgid "Service graphs"
+msgstr "Graphiques de services"
```
Comment (Thebarda) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16227]:
```suggestion
msgstr "Graphiques de service"
```

File: www/front_src/src/Resources/Details/index.test.tsx [hunk 3]
```diff
@@ -614,6 +621,11 @@ describe(Details, () => {
         setSelectedServiceResource();
       });
 
+      await waitFor(() => {
+        expect(mockedAxios.get).toHaveBeenCalledTimes(2);
+        expect(getByText(period) as HTMLElement).toBeEnabled();
+      });
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/index.test.tsx:627]: It's recommended to put one `expect` in a `waitFor`

File: www/front_src/src/Resources/Details/tabs/Graph/index.tsx [hunk 4]
```diff
@@ -65,13 +69,19 @@ const GraphTabContent = ({
...
     <GraphOptionsContext.Provider value={graphOptions}>
       <div className={classes.container}>
-        <TimePeriodButtonGroup />
-        <MousePositionContext.Provider value={mousePositionProps}>
-          <ExportablePerformanceGraphWithTimeline
-            graphHeight={280}
-            resource={details}
-          />
-        </MousePositionContext.Provider>
+        {details?.type === 'service' ? (
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Graph/index.tsx:72]:
```suggestion
        {equals(details?.type, 'service') ? (
```
It's more readable to extract it as a variable like `isService`

File: www/front_src/src/Resources/Details/tabs/Timeline/index.tsx [hunk 5]
```diff
@@ -29,8 +29,9 @@ type TimelineListing = ListingModel<TimelineEvent>;
 
 const useStyles = makeStyles((theme) => ({
   filter: {
+    // marginTop: theme.spacing(1),
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Timeline/index.tsx:32]: Is it going to be used ?

File: www/front_src/src/Resources/Details/tabs/index.tsx [hunk 6]
```diff
@@ -58,6 +58,10 @@ const tabs: Array<Tab> = [
         return false;
       }
 
+      if (details.type === 'host') {
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/index.tsx:61]:
```suggestion
      if (equals(details.type, 'host')){
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po (hunk 2 above)
Comment (bdauria) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16227]: I think it'd be more correect in french to talk about many services, in oppisition to english

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**